packages/block-library/src/separator/block.json

```diff
@@ -8,16 +8,18 @@
 	"keywords": [ "horizontal-line", "hr", "divider" ],
 	"textdomain": "default",
 	"attributes": {
-		"color": {
-			"type": "string"
-		},
-		"customColor": {
+		"opacity": {
 			"type": "string"
 		}
 	},
 	"supports": {
 		"anchor": true,
-		"align": [ "center", "wide", "full" ]
+		"align": [ "center", "wide", "full" ],
+		"color": {
```

Comment:
Should we add a default control here?

```json
			"__experimentalDefaultControls": {
				"background": true
			}
```

Without it the panel is closed:

<img width="275" alt="Screen Shot 2022-02-24 at 9 24 42 am" src="https://user-images.githubusercontent.com/6458278/155419416-c4997244-0436-4cbd-9348-4dc14159d62d.png">

Reply:
Good point, done.